### PR TITLE
Add metadata.yml & fix typo

### PR DESCRIPTION
**Pull request by acka47**
Hi everybody, hbz and TIB are jointly developing and operating the [OER Search Index (OERSI)](https://oersi.org). I thought the Annif tutorial would be a good resource to be findable in OERSI. There is a [workflow to include a GitHub repo in OERSI](https://oersi.org/resources/pages/de/docs/sources/gitlab_github/). If you merge this PR and additionally add the topic "open-educational-resources" to the repo, OERSI should automatically start indexing the tutorial.

### CODE REVIEW

File: metadata.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    type: Person
+    affiliation:
+      name: National Library of Finland
+      type: Organization
+  - givenName: Mona
+    familyName: Lehtinen
+    id: https://orcid.org/0009-0004-1260-823X
+    type: Person
+    affiliation:
+      name: University of Turku
```
Comment (juhoinkinen) [inline metadata.yml:36]: University of Turku is not correct, should be "National Library of Finland" like for me and Osma. FYI @monalehtinen

Comment (monalehtinen) [reply metadata.yml:36]: Also orcid should be https://orcid.org/0000-0002-4735-0214

Comment (acka47) [reply metadata.yml]: My apologies, I must have gotten the wrong profile. Fixed with 57a764e87b718328e0d7a467c9fafb75c8912c85

**Comment by acka47** [comment]
Feeld free to squash when merging...l

**Comment by acka47** [comment]
FYI, the tutorial has now automatically been added to OERSI: https://oersi.org/resources/aHR0cHM6Ly9naXRodWIuY29tL05hdExpYkZpL0FubmlmLXR1dG9yaWFs

**Pull request merged**